src/force-latest-compat.jl

```diff
@@ -0,0 +1,47 @@
+using TOML
+
+function latest_compat(range::VersionRange)
+    lower = range.lower
+    upper = range.upper
+    l_t = Int.(lower.t[1:lower.n])
+    l_ver = VersionNumber(l_t)
+    u_t = Int.(upper.t[1:upper.n])
+    u_ver = VersionNumber(u_t)
+    if l_ver > u_ver
+        return l_t, l_ver
+    else
+        return u_t, u_ver
+    end
+end
+
+function latest_compat(s::VersionSpec)
+    x = latest_compat.(s.ranges)
+    return join(string.(x[argmax(getindex.(x, 2))][1]), ".")::String
```

Comment:
This is quite hard to understand what it does.

Reply:
Agreed.

I'll break it up into multiple lines to make it clearer what's going on.

Reply:
Alright, take a look now.

Reply:
I think this is resolved now. The new version of the function splits things up into multiple lines, and should be much more readable.

Let me know if this is not resolved.